src/components/views/dialogs/security/AccessSecretStorageDialog.tsx

```diff
@@ -343,6 +343,14 @@ export default class AccessSecretStorageDialog extends React.PureComponent<IProp
                 )}</p>
 
                 <form className="mx_AccessSecretStorageDialog_primaryContainer" onSubmit={this.onPassPhraseNext}>
+                    <input
+                            className="mx_AccessSecretStorageDialog_hidden-encryption-username-for-auto-fill-password"
+                            name="username" // make it a little easier for browser's remember-password
+                            key="username_input"
+                            type="text"
+                            placeholder={_t("Username").toLocaleLowerCase()}
+                            value="Passphrase"
```

Comment:
this'll conflict between your various sessions/credentials

Reply:
Yes it will but it's better than conflicting between your username like it does now, no? I think for most users it'll be better to not have the username and then the passphrase because that'll confuse them and if they really want to these fields can be edited anyway for power users (which you are usually when you have a few accounts) but if it's a must i can add the username to the end of the value e.g. passphrase of @user:domain.com or something similar